src/PSS/RequestClient.php

```diff
@@ -10,6 +10,16 @@ class RequestClient extends BaseClient
 {
     protected $basePath = 'pss/';
 
+    protected $requestMap = [
```

Comment:
why cant we just define this in entity

Reply:
I think we want to move all API specific knowledge out of the entity classes

/cc @gavtaylor

Reply:
How would this look if one client interacted with multiple entities?

Reply:
You'd have multiple map properties

Reply:
We could move this into the Entity as a constant, I guess. Main issue of doing it this way is if an entity is used in multiple clients